.gitignore

```diff
@@ -7,8 +7,30 @@ default/
 # Mac stuff
 *.DS_Store
 
-# IDE files
+# QT Creator
+*.user
```

Comment:
If we're going to ignore `*.user`, then a bunch of ignore rules below are not needed.